bitcoin/src/blockdata/opcodes.rs

```diff
@@ -29,10 +29,13 @@ pub struct All {
 // private import so we don't have to use `all::OP_FOO` in this file.
 use self::all::*;
 
+/// Enables wildcard imports to bring into scope all opcodes and nothing else.
+///
 /// The `all` module is provided so one can use a wildcard import `use bitcoin::opcodes::all::*` to
 /// get all the `OP_FOO` opcodes without getting other types defined in `opcodes` (e.g. `All`, `Class`).
+///
+/// This module is guaranteed to never contain anything except opcode constants.
```

Comment:
And all opcode constants are guaranteed to begin with `OP_`.

Reply:
Used as suggested.